pep_sphinx_extensions/pep_theme/static/style.css

```
@@ -225,14 +226,14 @@ div.table-wrapper {
 table {
     width: 100%;
     border-collapse: collapse;
-    border-top: 1px solid var(--colour-background-accent);
-    border-bottom: 1px solid var(--colour-background-accent);
+    border-top: 1px solid var(--colour-background-accent-light);
+    border-bottom: 1px solid var(--colour-background-accent-light);
 }
 table caption {
     margin: 1rem 0 .75rem;
 }
```

```suggestion
}
table tr {
    border-bottom: 1px solid var(--colour-rule-strong);
}
table thead tr {
    background-color: var(--colour-background-accent-strong);
}
```

Add horizontal rules and clearly delineate header row